packages/meditrak-server/src/apiV2/requestPasswordReset.js

```diff
@@ -28,7 +28,8 @@ export const requestPasswordReset = async (req, res) => {
     user_id: user.id,
   });
 
-  const resetUrl = process.env.PASSWORD_RESET_URL.replace('{token}', token);
+  const passwordResetUrl = `${process.env.TUPAIA_FRONT_END_URL}/reset-password?passwordResetToken={token}`;
```

Comment:
Replace PASSWORD_RESET_URL  and VERIFY_EMAIL_URL  with TUPAIA_FRONT_END_URL to save duplication.